tests/data/expression.diff

```diff
@@ -116,14 +128,16 @@
  call(**self.screen_kwargs)
  call(b, **self.screen_kwargs)
  lukasz.langa.pl
-@@ -93,23 +114,25 @@
+@@ -93,23 +126,27 @@
  1.0 .real
  ....__class__
  list[str]
  dict[str, int]
  tuple[str, ...]
 -tuple[str, int, float, dict[str, int],]
-+tuple[str, int, float, dict[str, int]]
++tuple[
++    str, int, float, dict[str, int],
++]
```

Comment:
Isn’t this a bug?

Reply:
No, this is literally the point of this change. You're looking at a diff-of-diffs, maybe that's the confusing bit?

Reply:
Yes this is confusing.

Is black going to change one-line typing declarations to be on two lines?

Reply:
If the input has a trailing comma (as in this example), yes.